src/canvas-view/canvas-editor/canvas-editor.less

```diff
@@ -0,0 +1,194 @@
+.canvas-editor-container {
+  background-color: @color-pf-white;
+  height: 100%;
+  width: 100%;
+  .canvas-editor-toolbar {
+    padding: 15px;
+    a {
+      color: @color-pf-black-600;
+      cursor: pointer;
+    }
+    a.disabled, a:hover.disabled  {
+      color: @color-pf-black-400;
+      cursor: not-allowed;
+    }
+    a:hover {
+      color: @color-pf-black-800;
+    }
+    button {
+      margin-right: 6px;
+    }
+    .more-actions {
+      border-left: solid 2px @color-pf-black-300;
+      margin-left: 6px;
+      vertical-align: middle;
+    }
+    .pficon {
+      font-size: 20px;
+      margin-left: 15px;
+      margin-top: -4px;
+      vertical-align: middle;
+    }
+    .right-aligned-controls {
+      display: block;
+      float: right;
+      font-size: 12px;
+      font-weight: 600;
+      padding-right: 5px;
+      padding-top: 2px;
+      vertical-align: middle;
+    }
+    .show-hide-connectors-label {
+      vertical-align: 2px;
+    }
+  }
+  .canvas-editor-toolbox-container {
+    -moz-placeholder-font-style: italic;
+    -moz-placeholder-padding-left: 10px;
+    -ms-input-placeholder-font-style: italic;
+    -ms-input-placeholder-padding-left: 10px;
+    height: 100%;
+    position: relative;
+    width: 100%;
+    .canvas-editor-toolbox {
+      background-color: rgba(255, 255, 255, 0.94);
```

Comment:
No, I'm talking way back to Colleen's original InVision mockups. I'm not sure the current design you mention above actually shows nodes behind the toolbox, and that they are hidden by the toolbox.

Either way is fine with me, easy enough to change the value :-)